packages/block-library/src/blocks/slideshow/init.php

```diff
@@ -84,69 +93,27 @@ class="<?php echo esc_attr( join( ' ', $classes ) ); ?>"
             <div class="novablocks-slideshow__slider">
-				<?php foreach ( $attributes['galleryImages'] as $media ) {
-
-                    $thisMediaStyle = $mediaStyle;
-
-                    if ( ! empty( $media['focalPoint'] ) ) {
-                        $thisMediaStyle = $thisMediaStyle . novablocks_get_focal_point_style( $media['focalPoint'] );
-                    } ?>
-
-                    <div class="novablocks-slideshow__slide">
-                        <div class="novablocks-slideshow__slide-wrap">
-	                        <div class="novablocks-slideshow__background novablocks-u-background">
-								<div class="novablocks-mask">
-									<div class="novablocks-parallax">
-
-										<?php
-										if ( $media['type'] === 'image' && ! empty( $media['url'] ) ) {
-											$id = attachment_url_to_postid( $media['url'] );
-											$width = $media['sizes']['full']['width'];
-											$height = $media['sizes']['full']['height'];
-
-											if ( ! empty( $id ) ) {
-												echo wp_get_attachment_image( $id, 'novablocks_huge', false, array(
-													'class' => 'novablocks-slideshow__media',
-													'style' => esc_attr( $thisMediaStyle ),
-													'data-width' => esc_attr( $width ),
-				                                    'data-height' => esc_attr( $height )
-												) );
-											} else {
-
-												$image_url = novablocks_get_image_url($media, 'novablocks_large');
-
-											    ?>
-											<img class="novablocks-slideshow__media"
-											     src="<?php echo esc_url( $image_url ); ?>"
-											     style="<?php echo esc_attr( $thisMediaStyle ); ?>"
-											     data-width="<?php echo esc_attr( $width ); ?>"
-											     data-height="<?php echo esc_attr( $height ); ?>"
-											/>
-											<?php }
-										} ?>
-
-										<?php if ( 'video' === $media['type'] ) { ?>
-											<video class="novablocks-slideshow__media" muted autoplay playsInline loop
-											       src="<?php echo esc_url( $media['url'] ); ?>"
-											       style="<?php echo esc_attr( $thisMediaStyle ); ?>"
-											       data-width="<?php echo esc_attr( $width ); ?>"
-											       data-height="<?php echo esc_attr( $height ); ?>"/>
-										<?php } ?>
-
-									</div>
-								</div>
-	                        </div>
-	                        <div class="novablocks-slideshow__foreground novablocks-foreground novablocks-u-content-padding novablocks-u-content-align">
-                                <div class="novablocks-slideshow__inner-container novablocks-u-content-width">
-                                    <?php
-                                    novablocks_the_media_title( $media, '<h2>', '</h2>' );
-                                    novablocks_the_media_caption( $media );
-                                    ?>
-                                </div>
-	                        </div>
-                        </div>
-                    </div>
-				<?php } ?>
+
```

Comment:
There should be a check for when there are no slides and not output and / or initialize the slider in this case.